pkgs/development/libraries/cosmopolitan/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenv, makeWrapper, runCommand, cosmopolitan }:
+stdenv.mkDerivation {
+  pname = "cosmopolitan-unstable";
+  version = "20210228";
+  src = builtins.fetchGit {
```

Comment:
`builtins` fetchers should not be used to fetch sources because they fetch at evaluation time. Please use `fetchgit` or `fetchFromGitHub`.